assets/js/timetree.js

```diff
@@ -0,0 +1,964 @@
+          .attr("width", 60)
+          .attr("height", 38)
+      );
+
+    // determine placement for branches left to right
+    // NOTE: could this be construed as an axis of some kind?
+    this.branchCoords = {};
+    let branchMargin = 100;
+    // etermine how much space to give to each branch
+    let branchWidth = (width - branchMargin * 2) / 5;
+    // calculate the midpoint of each branch and set for easy lookup
+    for (const [i, b] of Object.keys(branches).entries()) {
+      this.branchCoords[b] =
+        branchMargin + min_x + i * branchWidth + branchWidth / 2;
+    }
+
+    this.trunkTop = this.yScale(this.leafStats.minYear - 15);
+    drawTrunk(
+      background,
+      [this.min_x, this.min_y, this.width, this.height],
+      this.trunkTop
+    );
+
+    // for debugging: mark the center of the svg
+    // svg
+    //   .append("circle")
+    //   .attr("r", 5)
+    //   .attr("fill", "red")
+    //   .attr("cx", this.min_x + this.width / 2)
+    //   .attr("cy", this.min_y + this.height / 2);*/
+
+    // for debugging: mark the center bottom of the svg
+    // this.svg
+    //   .append("circle")
+    //   .attr("r", 5)
+    //   .attr("fill", "red")
+    //   .attr("cx", 0)
+    //   .attr("cy", this.min_y + this.height);
+
+    // for debugging: mark the bounds of the svg
+    // this.svg
+    //   .append("rect")
+    //   .attr("x", this.min_x)
+    //   .attr("y", this.min_y)
+    //   .attr("width", this.width)
+    //   .attr("height", this.height)
+    //   .attr("stroke", "red")
+    //   .attr("fill", "none");
+
+    let simulation = d3
+      .forceSimulation(this.network.nodes)
+      .force("charge", d3.forceManyBody().strength(forceStrength.charge))
+      // .alpha(0.1)
+      .alphaDecay(0.2)
+      .force(
+        "collide",
+        d3.forceCollide().radius((d) => {
+          // collision radius varies by node type
+          if (d.type == "leaf") {
+            return leafSize.width; // - 5;
+          }
+          return 2;
+        })
+      )
+      .force(
+        "link",
+        d3.forceLink(this.network.links).strength((link) => {
+          return link.value; // link strength defined when links created
+        })
+      )
+      .force(
+        "y",
+        d3
+          .forceY()
+          .y((node) => this.centuryY(node))
+          .strength(forceStrength.centuryY)
+      )
+      .force(
+        "x",
+        d3
+          .forceX()
+          .x((node) => this.branchX(node))
+          .strength((node) => {
+            if (node.century != undefined) {
+              // apply the force more strongly the further up the tree we go
+              return forceStrength.branchX * (node.century - 14);
+            }
+            return 0;
+          })
+      )
+      .force("bounds", this.leafBounds.bind(this));
+
+    // run simulation for 30 ticks without animation to set positions
+    // (30 is enough with alpha decay of 0.2; need 300 with default alpha decay)
+    simulation.tick(30);
```

Comment:
I have done this in another project as well where I wanted to bypass the animation. So I think this is the way to go. I had many more nodes so the loading time was longer, however, and without the animation it's important to give the user some feedback. It seems like this isn't a problem for the current size of the tree, but this is a place to keep an eye on if the data will grow.